draft-ietf-sframe-enc.md

```diff
@@ -875,6 +875,70 @@ The authors wish to specially thank Dr. Alex Gouaillard as one of the early
 contributors to the document. His passion and energy were key to the design and
 development of SFrame.
 
+# Example API
+
+**This section is not normative.**
+
+This section describes a notional API that an SFrame implementation might
+expose.  The core concept is an "SFrame context", within which KID values are
+meaningful.  In the key management scheme described in {{sender-keys}}, each
+sender has a different context; in the scheme described in {{mls}}, all senders
+share the same context.
+
+An SFrame context stores mappings from KID values to "key contexts", which are
+different depending on whether the KID is to be used for sending or receiving
+(an SFrame keys should never be used for both operations).  A key context tracks
+the key and salt associated to the KID, and the current CTR value.  A key
```

Comment:
```suggestion
the key and salt associated to the KID, and the current CTR value.  The key
```
It makes sense as is. Whichever you prefer.

Reply:
Nah, this is "A context to be" as in "Any context which is to be..."